ang/civicase/CaseDetails--tabs--people.html

```diff
@@ -0,0 +1,284 @@
+        </select>        
+      </div>
+    </div>
+    <div class="civicase__people-tab__filter">
+      <paging
+      class="center-block paging-top"
+      page="rolesPage"
+      page-size="25"
+      total="rolesCount"
+      ng-show="rolesCount > 25"
+      show-prev-next="true"
+      show-first-last="true"
+      text-first="{{ ts('<< First') }}"
+      text-last="{{ ts('Last >>') }}"
+      text-next="{{ ts('Next >') }}"
+      text-prev="{{ ts('< Prev') }}"
+      >
+      </paging>
+      <div>
+        <a ng-repeat="a in letters" class="civicase__people-tab__filter-alpha-pager" ng-class="{active: a === rolesAlphaFilter}" ng-click="setLetterFilter(a, 'roles')">{{ a }}</a>
+        <a href class="all" ng-click="setLetterFilter(undefined, 'roles')">All</a>
+      </div>      
+      <div class="civicase__people-tab__filter-role">
+        <select class="form-control" ng-model="rolesFilter" crm-ui-select>
+          <option value="">{{ ts('All Roles') }}</option>
+          <option value="client">{{ ts('Client') }}</option>
+          <option ng-repeat="role in allRoles" value="{{ role.name }}">{{ role.role }}</option>
+        </select>
+      </div>    
+    </div>
+    <div>
+    </div>
+    <table class="civicase__people-tab__table table">
+      <thead class="civicase__people-tab__table-body">
+        <tr>
+          <th class="civicase__people-tab__table-column">{{ ts('Role') }}</th>
+          <th class="civicase__people-tab__table-column">{{ ts('Name') }}</th>
+          <th class="civicase__people-tab__table-column">{{ ts('Start date') }}</th>
+          <th class="civicase__people-tab__table-column">{{ ts('Phone') }}</th>
+          <th class="civicase__people-tab__table-column">{{ ts('Email') }}</th>
+          <th class="civicase__people-tab__table-column civicase__people-tab__table-column--last"></th>
+        </tr>
+      </thead>
+      <tbody class="civicase__people-tab__table-body">
+        <tr ng-repeat="role in roles" ng-class="{unassigned: !role.contact_id}">
+          <td class="civicase__people-tab__table-column civicase__people-tab__table-column--first">
+            <span ng-if="role.contact_id" class="civicase__checkbox" ng-if="!$parent.viewingCase && $parent.selectedCases && $parent.bulkAllowed" role="checkbox" aria-checked="{{activity.is_completed}}" tabindex="0" aria-labelledby="Mark complete/incomplete">
+              <input id="select-role-{{ $index }}" class="civicase__people-tab__table-checkbox" type="checkbox" ng-model="role.checked" ng-click="setSelectionMode('checked', 'roles')" />
+              <i ng-if="role.checked" class="civicase__checkbox--checked material-icons">check_box</i>
+            </span>
+            <label for="select-role-{{ $index }}">
+              <div><strong>{{ role.role }}</strong></div>
+              <div><em>{{ role.description }}</em></div>
+              <div>{{ role.desc }}</div>
```

Comment:
what is the difference between `role.description` and `role.desc` ?

Reply:
Haven't written this. I think the role description is the description of the role and role.desc is the extra description that user fills when they update the assignee.